vnext/src/Libraries/Image/Image.windows.js

```diff
@@ -1,10 +1,13 @@
 /**
- * Copyright (c) Microsoft Corporation.
- * Licensed under the MIT License.
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
This file was marked as a copy-style override, so any changes will get clobbered during upgrades. Adding validation to prevent tampering discovered it wasn't a verbatim copy of the original file, so I needed to restore this as an exact copy.